numerics/cbrt.cpp

```diff
@@ -0,0 +1,71 @@
+﻿
+#include "numerics/cbrt.hpp"
+
+#include <pmmintrin.h>
+
+#include <cstdint>
+#include <limits>
+
+namespace principia {
+namespace numerics {
+
+constexpr std::uint64_t C = 0x2A9F7893782DA1CE;
+static const __m128d sign_bit =
+    _mm_castsi128_pd(_mm_cvtsi64_si128(0x8000'0000'0000'0000));
```

Comment:
In other places we condition the use of intrinsics on some preprocessor symbol and use a slow path otherwise.  Should we do the same in this function?